### ISSUE TITLE
Choose photo button and checkbox labels have no focus styles

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#5567]
If navigating with a keyboard in the edit user profile page, there's no focus styles for the user "Choose photo" upload button:

<img width="352" alt="user_profile_for_muffinresearch__script_alert_0___script__ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42584641-8eac7f0a-852b-11e8-87bb-eb81fa657417.png">

The checkboxes similarly have no focus styles. If using a keyboard to navigate there's no way of knowing that you're on a checkbox to be able to interact with it.

<img width="734" alt="user_profile_for_muffinresearch__script_alert_0___script__ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42584786-e78084be-852b-11e8-8beb-95bfc024e9a1.png">

**Comment by ioanarusiczki** [comment]
Verified fixed on AMO dev with FF61(Win10)

![navigating using keyboard](https://user-images.githubusercontent.com/33448286/42816027-015a3812-89d2-11e8-914d-a26d2563a51d.gif)
